test/SlowTests/Issues/RavenDB_22498.cs

```diff
@@ -0,0 +1,270 @@
+﻿using System;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using FastTests;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Documents.Operations.Indexes;
+using Raven.Client.Http;
+using Raven.Client.Json.Serialization;
+using Raven.Client.ServerWide.Operations;
+using Raven.Server.Documents.Handlers.Processors.Indexes;
+using Sparrow.Json;
+using Tests.Infrastructure;
+using Tests.Infrastructure.Entities;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues;
+
+public class RavenDB_22498 : RavenTestBase
```

Comment:
Maybe we could test group by using complex field too?

Reply:
I'm pretty sure it is part of the slow or fast tests